api/src/main/java/org/pmiops/workbench/api/BugReportController.java

```diff
@@ -0,0 +1,48 @@
+package org.pmiops.workbench.api;
+
+import javax.mail.Message;
+import javax.mail.Message.RecipientType;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.MimeMessage;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.AddressException;
+import java.io.UnsupportedEncodingException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.Properties;
+import org.pmiops.workbench.firecloud.ApiException;
+import org.pmiops.workbench.model.BugReport;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class BugReportController implements BugsApiDelegate {
+
+  private static final Logger log = Logger.getLogger(BugReportController.class.getName());
+
+
+
+  @Override
+  public ResponseEntity<BugReport> sendBug(BugReport bugReport) {
+    Properties props = new Properties();
+    Session session = Session.getDefaultInstance(props, null);
+    try {
+      Message msg = new MimeMessage(session);
+      // To test the bug reporting functionality, change the recipient email to your email rather than the group.
```

Comment:
Add a TODO with reference to JIRA ticket for making these groups configurable per-environment, and to add the user's contact e-mail into the body of the e-mail that is sent out.